deploy/olm-catalog/ibm-commonui-operator/1.2.3/ibm-commonui-operator.v1.2.3.clusterserviceversion.yaml

```diff
@@ -282,9 +282,9 @@ spec:
                 - name: OPERATOR_NAME
                   value: ibm-commonui-operator
                 - name: COMMON_WEB_UI_IMAGE_TAG_OR_SHA
-                  value: 1.2.2
+                  value: sha256:d5fda83f0abd6deb8c484338d093255d32b29a3adde7c15300a695e9ec6a9e00
                 - name: LEGACYHEADER_IMAGE_TAG_OR_SHA
-                  value: sha256:35a27f3b01569b41668d7033999681a269d274d586a7948cfc3e2dc024a310e0
+                  value: sha256:5c785b6c4dc2b53af8e0219415388e4bafcfce354c13c6ff62912a9e7c3abb46
                 image: quay.io/opencloudio/ibm-commonui-operator:latest
```

Comment:
@ericabr Is there any specific sha image that we need to use for our operator?

Reply:
just added it :)